### ISSUE TITLE
Prefix of player (from LuckPerms) not showing up?

### ISSUE THREAD

**Issue by d1axowy-eulerion** [montlikadani/TabList#355]
### Type of bug

Other

### TabList version

[20:11:34 INFO]: TabList version 5.6.3 [20:11:34 INFO]: An ultimate animated tablist [20:11:34 INFO]: Author: montlikadani

### Software version

[20:11:49 INFO]: This server is running CraftBukkit version git-Spigot-21fe707-e1ebe52 (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT) [20:11:49 INFO]: You are 52 version(s) behind

### TabList configuration files

https://pastebin.com/fJAh5KNb

### Bug description

Basically when I click TAB, everything's working, except it doesn't show the player's prefix
In the attachment, the name 
![2022-06-23_20 14 09](https://user-images.githubusercontent.com/105602444/175366771-b57db842-533f-4f9f-96de-b68935f4e127.png)

**Comment by montlikadani** [comment]
You don't have enabled `change-prefix-suffix-in-tablist` in config, and follow the documentation instead of asking such basic things: https://github.com/montlikadani/TabList/wiki/How-to-create-a-group

**Comment by d1axowy** [comment]
Thank you and sorry!
